docs/content/development/router.md

```diff
@@ -76,6 +76,8 @@ The router module uses several of the default modules in [Configuration](../conf
 |`druid.router.pollPeriod`|Any ISO8601 duration.|How often to poll for new rules.|PT1M|
 |`druid.router.strategies`|An ordered JSON array of objects.|All custom strategies to use for routing.|[{"type":"timeBoundary"},{"type":"priority"}]|
 |`druid.router.avatica.balancer.type`|String representing an AvaticaConnectionBalancer name|Class to use for balancing Avatica queries across brokers|rendezvousHash|
+|`druid.router.http.maxRequestBufferSize`|Maximum size of the buffer used to write requests when forwarding them to the broker.|8 * 1024|
```

Comment:
how does bufferSize relate to possible headerSize ?

Reply:
the bufferSize is used by the JettyHttpClient to write the request to when sending it to broker. This should be able to hold the entire request including the header. In relation to the headerSize, this should be set to atleast the maxHeaderSize allowed on the broker.

Reply:
we should probably mention the part, "this should be set to atleast the maxHeaderSize allowed on the broker", in the doc

Reply:
added.